client/src/actions/index.ts

```diff
@@ -118,6 +123,35 @@ async function datasetMetadataFetchAndLoad(
   }
 }
 
+/**
+ * Fetches and loads dataset spatial metadata.
+ * @param dispatch - Function facilitating update of store.
+ * @param oldPrefix - API prefix with dataset path that dataset metadata lives on. (Not S3 URI)
+ */
+async function datasetSpatialMetadataFetchAndLoad(
+  dispatch: AppDispatch,
+  oldPrefix: string
+): Promise<void> {
+  // TODO: remove testDatasetUrl post schema 5.1 migration
+  // temporary workaround to fetch spatial metadata from the test dataset
+  const testDatasetUrl = `${globals.API?.prefix}`;
```

Comment:
This is where we should plug the direct S3 URI fro the example dataset for the UXR

Reply:
Ah I think @seve was saying that we can just manually upload your mocked spacial dataset to S3 and Explorer dev/staging/prod will just load the mocked dataset from S3 like it currently does for any regular dataset (since all datasets are in S3)

In other words, it "feels" like we don't need special logics in the code to handle the mocked dataset, since Explorer already knows how to load dataset from S3

@seve please help confirm 🙏 

Thank you!

Reply:
ok sounds good - I'll remove this then